input/data/2009/2009A%26A...507..389A/info.yaml

```diff
@@ -0,0 +1,10 @@
+reference_id: 2009A&A...507..389A
+source_id: [60]
+telescope: [hess]
+
+datasets:
+  - tev-000060.yaml
+
+data_entry:
+  status: complete
```

Comment:
Please put the data entry status as incomplete. In table 3 there are spectra given for three time periods, but here you only do data entry for the total. I think that's fine, your time is better spent on scripting for gamma-cat, but just change the data entry status value.